my-app/src/components/feature/review/ReviewForm.tsx

```diff
@@ -0,0 +1,60 @@
+import { StarRating } from '@/shared/components/StarRating';
+import { generateRandomEmailAddress } from '@/shared/utils/random-email.generator';
+import { Review } from '@/types/review.type';
+import { Box, Button, Card, CardBody, Input } from '@chakra-ui/react';
+import { FormEvent, useState } from 'react';
+
+export type ReviewFormProps = {
+    addShowReview: (review: Review) => void;
+};
+
+export const ReviewForm = ({ addShowReview }: ReviewFormProps) => {
+    const [rating, setRating] = useState<number>(0);
+
+    const onSubmitHandler = (event: FormEvent) => {
+        event.preventDefault();
+        const reviewInputElement = document.getElementById(
+            'review-input'
+        ) as HTMLFormElement;
+
+        const value = reviewInputElement.value;
+
+        const newReview: Review = {
+            id: crypto.randomUUID().toString(),
+            email: generateRandomEmailAddress(),
+            review: value,
+            rating,
+        };
+
+        addShowReview(newReview);
+        setRating(0);
+        reviewInputElement.value = '';
+    };
+
+    return (
+        <Box
+            display="flex"
+            flexDir="column"
+            gap={4}
+            min-width="480px"
+            as="form"
+            onSubmit={(e: FormEvent) => onSubmitHandler(e)}
```

Comment:
Mozes ovo i direktno u ovom slucaju, s obzirom da ne mjenjas set props-a (argumenata) koje prosljeđuješ onSubmitHandler funkciji.

ukoliko bi ovdje radio nesto tipa
```
            onSubmit={(e) => onSubmitHandler(e, dodatni_argument)}
```
onda to ima smisla, ovako je nepotrebno :)

```suggestion
            onSubmit={onSubmitHandler}
```